src/scripts/todo.js

```diff
@@ -0,0 +1,44 @@
+const getStorageTodo = () => {
+    return (localStorage.getItem('todo'));
+}
+
+const addStorageTodo = (todo) => {
+    let todos = localStorage.getItem('todo');
+    if(!todos) todos = '';
+    todos += todos.length > 0 ? `,${todo}` : todo;
+    localStorage.setItem('todo',todos);
+}
+
+const addTodos = () => {
+    const container = document.getElementById('container');
+    const formContainer = document.createElement('div');
+    formContainer.id = 'todo-container';
```

Comment:
you could have the form container already in the DOM, and then just populate it. Could be my inexperience, but I'm not sure what the advantage is to populating the DOM after the page loads?